dll/win32/msi/msi.spec

```diff
@@ -294,3 +294,5 @@
 @ stdcall -private DllGetClassObject(ptr ptr ptr)
 @ stdcall -private DllRegisterServer()
 @ stdcall -private DllUnregisterServer()
+
+@ cdecl __wine_msi_call_dll_function(long ptr)
```

Comment:
Hmmm.... do we accept these kind of things? Is this function necessary? for what? If not, could we remove it?

Reply:
```
static DWORD CALLBACK custom_action_thread(void *arg)
{
    GUID guid = *(GUID *)arg;
    heap_free(arg);
    return __wine_msi_call_dll_function(client_pid, &guid);
}
```

Reply:
OK so basically this is (yet another example of) a piece of Wine hackish code... 😒😒

Reply:
yup

Reply:
at least they are starting to share code between modules now, instead of copy-pasting it every time...